NICE.Identity.Management/ClientApp/src/views/UsersList/__tests__/UsersList.test.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { mount, shallow } from "enzyme";
+import fetchMock from "fetch-mock";
+import toJson from "enzyme-to-json";
+import { UsersList } from "../UsersList";
+import users from "./users.json";
+import { MemoryRouter } from "react-router";
+import { nextTick } from "../../../utils/nextTick";
+
+afterEach(fetchMock.reset);
+
+it("should show loading message before data has been loaded", () => {
+	fetchMock.get("*", {});
+	const wrapper = shallow(<UsersList />);
+	expect(wrapper.find("p").text()).toEqual("Loading...");
+});
+
+it("should match the snapshot after data has been loaded", async () => {
```

Comment:
Should we have a test that's a bit more lower level that asserts that we load data in componentDidMount? At the moment, if we moved the data loading to the constructor the tests would still pass, but we probably want to specifically test that the componentDidMount lifecycle method is triggering a data load. Then a separate test to checks what happens when the data has loaded successfully.